MovingBlocks/TerasologyLauncher
# fix: use custom start scripts instead of brittle patch files

Okay, I've ended up going down the path suggested in #547 and adding custom templates based on the default start scripts provided by Gradle to this repo. 

They contain a minimal addition to set `JAVA_HOME` local to the `jre` folder in `APP_HOME`. That way, the local JRE should be picked up when starting the launcher.

### How to test

- `gradlew install...Dist` (with `...` being `Linux64`, `Windows64`, or `Mac`) to build the distribution for Windows
- start the launcher via `bin/TerasologyLauncher.bat` or `bin/TerasologyLauncher`
- download a game and start it
- check logs that it is using the local Java

Resolves #547

---

Original PR description:

~~The start scripts are generated by `gradlew startScripts`, and the patch logic hooks up to it via `doLast`. I've updated the _patch_ files by adjusting the `.orig` files as required, and then running `diff -u` on it (from within `build/scripts/`).~~

~~After this update, I at least don't see any rejections `.rej` anymore.~~
